src/vtok_agent/src/agent/mngtok.rs

```diff
@@ -106,6 +106,7 @@ impl DbSource {
                 id: 1,
                 label: "acm-key".to_string(),
                 encrypted_pem_b64: db.encrypted_private_key.clone(),
+                cert_pem: Some(self.cert_pem().unwrap().to_string()),
```

Comment:
This might break compatibility. This `unwrap()` should be changed. Thinking of a `if let Some(cert) = self.cert_pem()` else `None`. But this change we now send the certificate always when the agent is running? We also might have a `cert_chain_pem()` here.